.circleci/config.yml

```diff
@@ -62,6 +71,16 @@ jobs:
       - store_artifacts:
           path: ~/test-results/spec
 
+  test_alpine:
+    machine: true
+    environment:
+      <<: *env
+      TRAVIS_OS_NAME: linux
+      ARCH: x86_64-musl
+      ARCH_CMD: linux64
+      DOCKER_TEST_PREFIX: crystallang/crystal:ci-test-dev # TODO: Remove after 0.33.0
```

Comment:
I'm pushing a `0.32.1-alpine[-build]` with the changes of https://github.com/crystal-lang/distribution-scripts/pull/55 included. So this line can be removed and we can verify how it works.